examples/docker-deployment/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.9
+
+# install sparsezoo from github
+ARG SPARSEZOO_BRANCH=main
+RUN git clone https://github.com/neuralmagic/sparsezoo.git --depth 1 -b $SPARSEZOO_BRANCH
```

Comment:
why not pip install?

Reply:
Yeah probably should... Right now the server command below only works for 1.2, which isn't released on pypi yet. So pip install would fail currently. After 1.2 is released we can go back and update this to use pip install

Reply:
would suggest going with pip when branch is not set, otherwise from git using the given branch